docs/videos/live-app-review.md

```diff
@@ -202,6 +238,11 @@ sidebar_label: Application Reviews
   allowfullscreen>
 </iframe>
 
+> **We do a demo of creating a "guest" named account for an app where the gas fees are sponsored by a special app account**
+>
+> - The guest account doesn't exist (sometimes called a virtual or contract account) until the user upgrades, but the name is reserved because only the app is able to create "USERNAME.APP_NAME.near".
```

Comment:
```suggestion
> - The guest account doesn't exist (sometimes called a virtual or contract account) until the user upgrades, but the name is reserved because only the app is able to create `USERNAME.APP_NAME.near`.
```